config/prod.json

```diff
@@ -0,0 +1,21 @@
+{
+  "server": {
+    "port": 80,
```

Comment:
this port does not work on the rpi. Already used. Will we try to find the annoying process and this port or change it here ?

Reply:
I'll change `prod.json` for now. We'll try to find which program locks port 80 later.